### PR TITLE
Tao 4492 update message

### PR DESCRIPTION
**Pull request by Alroniks**
Customer provide a proper message for pop-up when a section was paused.

### CODE REVIEW

File: views/js/runner/plugins/security/sectionPause.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ define([
 ], function(_, __, pluginFactory, mapHelper) {
     'use strict';
 
-    var pauseMessage = __('Awaiting for the proctor authorization.');
+    var pauseMessage = __('All students will begin the next section at the same time. Please relax quietly until the room supervisor starts the next section.');
```
Comment (eacraft13) [inline views/js/runner/plugins/security/sectionPause.js]: May I suggest "Please patiently await for your test supervisor to start the next section."

Why...
- Will all students actually begin at the same time? Isn't it a feature to begin test takers at different times?
- 'Patiently' and 'relax quietly' are synonyms... I guess I just prefer 'patiently.'
- Do all students await in the same room? I think 'test supervisor' is better than 'room supervisor.'

**Comment by Alroniks** [comment]
Hi Eric. Text for this message provided by customer, so I guess it's already correct. And I can not merge own PRs, so I select you as you nearest person and you are in the same timezone :)

**Comment by eacraft13** [comment]
@Alroniks, sorry to push back again for something so simple... but this seems like it's placing brand specific content into generic code. Do we have a way to inject brand specific content into a plugin? Maybe through the test_runner_plugin_registry configuration file? Or the like?

**Comment by Alroniks** [comment]
@eacraft13 Let's invite somebody else for judging.
@krampstudio What do you think about this text? Should it be moved into config? As I know we don't have any configurations for these plugins.

**Comment by eacraft13** [comment]
So, my official stance is this (granted there is no simple solution to injecting brand specific configuration into a plugin)...

As this was requested by a customer, and that customer may be the only one currently using this plugin... I think, we accept this change as is (no need to do extra work yet). We create a think tank issue that enables injecting brand specific configuration into plugins (perhaps through a new `args` property in the test_runner_registry configuration file). And let the product owners decide when that feature is necessary.

**Comment by krampstudio** [comment]
We can push this plugin as it is. It's anyway so specific I don't really understand why it's under the common plugin repository.
It should have been an ACT plugin. 
If another one wants another message we will relay on a server config. But for now let's add this message.

**Comment by Alroniks** [comment]
@eacraft13 Do you agree? It would be nice to merge it today.

**Pull request merged**